### ISSUE TITLE
Docker setup - frontend container keeps exiting

### ISSUE THREAD

**Issue by maciejewiczow** [SFTtech/abrechnung#206]
I am trying to setup Abrechnung using the provided docker-compose files, with slight modification (namely adding a posgress db container). However the frontend container keeps exiting with code 1 and I can't see anything suspicious in the container logs. 
Logs:
```
/docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
/docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
/docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
10-listen-on-ipv6-by-default.sh: info: /etc/nginx/conf.d/default.conf differs from the packaged version
/docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
/docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
/docker-entrypoint.sh: Launching /docker-entrypoint.d/99-abrechnung.sh
```
Modified docker-compose:
```yml
services:
  api:
    extends:
      file: docker-compose.base.yml
      service: api
    image: quay.io/abrechnung/api:latest-release
    env_file: .env

  frontend:
    image: quay.io/abrechnung/frontend:latest-release
    depends_on:
      api:
        condition: service_healthy
    expose:
      - "80:2137"
    links:
      - api

  mailer:
    extends:
      file: docker-compose.base.yml
      service: mailer
    image: quay.io/abrechnung/api:latest-release
    links:
      - db
      - api

  cron:
    extends:
      file: docker-compose.base.yml
      service: cron
    image: quay.io/abrechnung/api:latest-release
    links:
      - api

  db:
    image: docker.io/postgres:14-alpine
    env_file: .env
    healthcheck:
      test: [ "CMD", "/usr/local/bin/pg_isready" ]
      start_period: 10s
    volumes:
      - "./data/pg:/var/lib/postgresql/data"

```
And my `.env` file:
```env
ABRECHNUNG_SERVICE__URL="http://localhost:8080"
ABRECHNUNG_SERVICE__API_URL="https://localhost:8080/api"
ABRECHNUNG_SERVICE__NAME="Abrechnung"
ABRECHNUNG_DATABASE__HOST="db"
ABRECHNUNG_DATABASE__USER="abrechnung"
ABRECHNUNG_DATABASE__DBNAME="abrechnung"
ABRECHNUNG_DATABASE__PASSWORD="[REDACTED]"

ABRECHNUNG_API__SECRET_KEY="[REDACTED]"
ABRECHNUNG_API__PORT=8080
ABRECHNUNG_API__ID="default"

ABRECHNUNG_REGISTRATION__ENABLED=false

ABRECHNUNG_EMAIL__ADDRESS=finansemieszkanie@gmail.com
ABRECHNUNG_EMAIL__HOST=imap.gmail.com
ABRECHNUNG_EMAIL__PORT=993
ABRECHNUNG_EMAIL__MODE=smtp-ssl # use this in production, remove line above
ABRECHNUNG_EMAIL__AUTH__USERNAME=finansemieszkanie@gmail.com
ABRECHNUNG_EMAIL__AUTH__PASSWORD='[REDACTED]'

POSTGRES_USER="abrechnung"
POSTGRES_PASSWORD="[REDACTED]" # use the same as DB_PASSWORD
POSTGRES_DATABASE="abrechnung" # use the same as DB_NAME
```
Any help would be greatly appreciated

**Comment by mikonse** [comment]
Hi,
it appears there still was an error in the startup of the frontend dockerfile.
It should be fixed in the newly released v0.13.2. Could you try again with that?

**Comment by maciejewiczow** [comment]
Okay, now everything seems to be working 😄 thanks for quick help
